include/robot_localization/ros_filter.h

```diff
@@ -476,6 +476,10 @@ template<class T> class RosFilter
     //!
     bool useControl_;
 
+    //! @brief Whether or not to print warning for tf lookup failure
+    //!
+    bool silent_tf_failure_;
```

Comment:
Thank you very much for both catching and addressing this issue.

Given that this is against `melodic-devel`, and we're still using the old (and technically incorrect) style for variable names in this branch, would you mind making this consistent with the others, and changing this to `silentTfFailure_`? I'll make sure it matches the style when I cherry pick it into the other branches. Thanks!

Reply:
My branch has been updated with this change.